test/univariate/continuous/johnsonsu.jl

```diff
@@ -0,0 +1,127 @@
+@testset "JohnsonSU" begin
+
+    d1 = JohnsonSU(0.0, 10.0, -2.0, 3.0)
+    @test d1 isa JohnsonSU{Float64}
+    @test params(d1) == (0.0, 10.0, -2.0, 3.0)
+    @test shape(d1) == 0.0
+    @test scale(d1) == 10.0
+    @test partype(d1) === Float64
+    @test eltype(d1) === Float64
+    @test rand(d1) isa Float64
+
+    @test median(d1) == quantile(d1, 0.5)
+    x = quantile.(d1, [0.25, 0.45, 0.60, 0.80, 0.90])
+    @test all(cdf.(d1, x) .≈ [0.25, 0.45, 0.60, 0.80, 0.90])
+    y = cquantile.(d1, [0.25, 0.45, 0.60, 0.80, 0.90])
+    @test all(ccdf.(d1, y) .≈ [0.25, 0.45, 0.60, 0.80, 0.90])
+
+    @test mean(d1) ≈ 7.581281
```

Comment:
Where did the reference values come from? Were they computed in R?

Reply:
I copied and pasted the values coming out of the implementation. I couldn't find similar functions in R. Is there any suggestion for alternatives?

Reply:
I've tested the values with MC in R (i.e. I ran `mean(rJohnsonSU(n=1000000, xi=0.0, lambda=10.0, gamma=-2.0, delta=3.0))` and `var(rJohnsonSU(n=1000000, xi=0.0, lambda=10.0, gamma=-2.0, delta=3.0))` and found the values are close to what's listed above). I also found kurtosis and skewness did't match but just decided to remove the functionality. (Sorry, no time for this project anymore 😕)